### PR TITLE
Creating Welcome page and user model

### PR DESCRIPTION
**Pull request by Emilianas**

### CODE REVIEW

**Comment by mjduarterosa** [comment]
Thanks Barbara! That's great :)

**Comment by Emilianas** [comment]
Barbara could you review and merge?

File: test/controllers/users_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
```
Comment (krissy) [inline test/controllers/users_controller_test.rb]: There's a new open issue for this so can address it separatly, but do we want to switch to Rspec for tests as it's a bit of an industry standard so good to get practice with?

**Comment by baisa** [comment]
Hi, it is now merged:) Sure we should talk about using Devise or similar gem, in case we decide to do it, we can always remove this User model

**Pull request merged**